src/engine/contactwriter.cpp

```diff
@@ -4745,9 +4803,14 @@ QContactManager::Error ContactWriter::syncUpdate(const QString &syncTarget,
                     QMap<QContactDetail::DetailType, DetailPair>::const_iterator cit = composedModifications.constBegin(), cend = composedModifications.constEnd();
                     for ( ; cit != cend; ++cit) {
                         // Apply these modifications to empty details, and add to the contact
+                        // XXX TODO: WHY? What is a "composed modification" ?
```

Comment:
Matt, can you explain this one?  Then I'll update the comment and remove the existing one.